shell/ShellApp/Views/MainPage.xaml

```diff
@@ -9,5 +9,6 @@
            LineBreakMode="WordWrap"
            Text="{Binding JwtPayload}" />
   </ScrollView>
+  
```

Comment:
you should not have more than 1 empty line in a row

Reply:
I have done this change in my request next commit.

Reply:
it appears that the multiple empty line has been replaced by a line with whitespace... please remove the whitespace